docs/submitting-games.md

```diff
@@ -0,0 +1,7 @@
+# Submitting Games
+
+When you've made a game that your proud of, you can submit it on [GitHub](https://github.com/Microsoft/pxt-arcade) for the chance to have it added to the homescreen.
+
+First, fork the [PXT Arcade](https://github.com/Microsoft/pxt-arcade) repository, and copy your game into a new file in the ``docs/community`` folder using either the [Blocks](/community/blocks-template) or [JavaScript](/community/javascript-template) template.
```

Comment:
We should give instructions that work from the web site. E.g. use the "create new file" button under ``/docs/community`` -- instead of having to use git directly.

Reply:
I'll see how a gif looks, might be able to do it even easier than I was thinking

Reply:
quick draft of an example gif, will probably be better to make it after the template and folder are in master here as this gif was made in my branch

![ezgif com-optimize (1)](https://user-images.githubusercontent.com/5615930/54569457-31bf7b80-4998-11e9-88d9-1970a0e15996.gif)